guides/plugins/plugins/administration/using-assets.md

```diff
@@ -0,0 +1,70 @@
+# Using assets
+
+## Overview
+
+When working with an own plugin, the usage of own custom images or other assets is a natural requirement. 
+So of course you can do that in Shopware as well. In this guide we will explore how you can
+add and use custom assets in your plugin in order to use them in the administration.
+
+## Prerequisites
+
+In order to be able to start with this guide, you need to have an own plugin running. As to most guides, this guide
+is also built upon the Plugin base guide:
+
+{% page-ref page="../plugin-base-guide.md" %}
+
+Needless to say, you should have your image or another asset at hand to work with.
+
+## Using custom assets
+
+In order to add your own custom assets, you need to save your assets in the `Resources/app/administration/static` folder.
+```bash
+# PluginRoot
+.
+├── composer.json
+└── src
+    ├── Resources
+    │   ├── app
+    │       └── administration
+    │             └── static
+    │                   └── your-image.png <-- Asset file here
+    └── SwagBasicExample.php
+```
+
+Similar as in [using custom assets in Storefront](./../storefront/add-custom-assets.md), you need to execute the 
+following command:
+```bash
+bin/console assets:install
+```
```

Comment:
``````suggestion
{% code %}
```bash
bin/console assets:install
```
{% endcode %}
``````

Reply:
Why though? The result will be the very same afaik